charts/newrelic-infrastructure/templates/controlplane/_enabled_helper.tpl

```diff
@@ -0,0 +1,62 @@
+{{/*
+Create ControlPlane Agent ConfigMap
+*/}}
+{{- define "nriKubernetes.controlPlane.agentConfigMap" -}}
```

Comment:
(Assuming that the answer to the first conversation is "yes")

I believe that this can be simplified into having just one helper function. Es:
`isControlPlaneNeeded = {{- if and (.Values.controlPlane.enabled) (not (include "newrelic.common.gkeAutopilot" .)) (not (include "newrelic.fargate" .)) -}}`

And then add a second if in the cases that are needed (RBACs/Service account)
```
if isControlPlaneNeeded{
   if {rbac.create} 
}
```

Reply:
That makes more sense.  I can update the helper if we land on this approach.